modules/phase_field/tests/rigidbodymotion/tests

```diff
@@ -1,49 +1,51 @@
 [Tests]
-  [./grain_center]
-    type = 'Exodiff'
-    input = 'grain_center.i'
-    exodiff = 'grain_center_out.e'
-  [../]
-
-  [./grain_force]
-    type = 'Exodiff'
-    input = 'grain_constforce.i'
-    exodiff = 'grain_constforce_out.e'
-  [../]
-
-  [./grain_advection]
-    type = 'Exodiff'
-    input = 'grain_advection_constforce.i'
-    exodiff = 'grain_advection_constforce_out.e'
-  [../]
 
   [./grain_motion]
     type = 'Exodiff'
     input = 'grain_motion.i'
     exodiff = 'grain_motion_out.e'
+    allow_warnings = true
   [../]
 
   [./grain_forcedensity]
     type = 'Exodiff'
     input = 'grain_forcedensity.i'
     exodiff = 'grain_forcedensity_out.e'
+    allow_warnings = true
+  [../]
+
+  [./grain_motion2]
+    type = 'Exodiff'
+    input = 'grain_motion2.i'
+    exodiff = 'grain_motion2_out.e'
+    allow_warnings = true
```

Comment:
We have warnings in `NonlocalKernel `and `ShapeElementUserObject` about the nonlocal jacobian being computationally expensive. That would reflect in these tests as well.